src/renwindow.h

```diff
@@ -6,19 +6,17 @@ struct RenWindow {
 #ifdef LITE_USE_SDL_RENDERER
   SDL_Renderer *renderer;
   SDL_Texture *texture;
-  SDL_Surface *surface;
-  int surface_scale;
+  RenSurface rs;
```

Comment:
wouldn't the name `surface` be better to keep?
The previous type was a pointer, no wrong structure dereferencing should cause an invalid compile due here.

Reply:
> It's mostly to distinguish it from the SDL surface underneath, so we avoid `RenWindow.surface.surface`.
fair.

But I feel like the name `rs` is a bit undescriptive.

> What do you mean?

Ignore it. Just Connecting dots that don't need connecting

Reply:
> But I feel like the name `rs` is a bit undescriptive.

Yeah that's fair.
What name should we use? We could even change the name of the underlying SDL_Surface.

Reply:
I think it's fine; but if you must, you could simply use `rensurface`; that's extremely clear, and falls in line with the normal coding practice of capitalizing the class, and lower-casing the instance.